cmd/kops/create_cluster.go

```diff
@@ -270,6 +273,8 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringVar(&options.MasterTenancy, "master-tenancy", options.MasterTenancy, "The tenancy of the master group on AWS. Can either be default or dedicated.")
 	cmd.Flags().StringVar(&options.NodeTenancy, "node-tenancy", options.NodeTenancy, "The tenancy of the node group on AWS. Can be either default or dedicated.")
 
+	cmd.Flags().StringVar(&options.APILoadbalancer, "api-loadbalancer", options.APILoadbalancer, "Sets the API loadbalancer to either 'public' or 'internal'")
```

Comment:
The field here is `API.LoadBalancer.Type`.  How about we make the flag `api-loadbalancer-type`?  It makes the flag more self documenting, and has a nice symmetry IMO.

(And then it doesn't really matter, because it is not user facing, but we could name the option `APILoadbalancerType` or `APILoadBalancerType`)